src/core/InputDropdown/style.ts

```diff
@@ -271,3 +282,12 @@ export const StyledCounter = styled("span")`
   `;
   }}
 `;
+
+export const StyledPopper = styled(Popper)`
+  & {
+    .${autocompleteClasses.popperDisablePortal} {
+      position: relative !important;
```

Comment:
same here re: important

Reply:
This popper is just used in the story and not a part of the component library and won't be used by consumers so I thought it would be ok, I can still look into it if not